cpp/models/abm/world.cpp

```diff
@@ -103,6 +101,19 @@ void World::migration(TimePoint t, TimeSpan dt)
             }
         }
     }
+    // check if a person makes a trip
+    size_t num_trips = m_migration_data.get_trips().size();
```

Comment:
since this is the only use of get_trips, maybe remove get_trips and add is_empty() to expose less of the internals of the class.

Reply:
In the function that follows, we need to know the number of trips to decide when to stop looking for new trips. I changed the function to return the number of trips instead.